test-extension/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "http://json.schemastore.org/chrome-manifest",
```

Comment:
move to `.vscode/settings.json`?

for chrome is syntax error

Reply:
Not syntax error, just a warning. And this extension is used for manually testing during the development so no need to change.